packages/containers/package.json

```diff
@@ -34,7 +34,7 @@
     "invariant": "^2.2.2",
     "keycode": "^2.2.0",
     "lodash": "^4.17.4",
-    "memoize-one": "^4.0.0",
+    "memoize-one": "^5.0.0",
```

Comment:
Bonus: align memoize-one version.
Breaking change is about the second parameter, that we don't use
```
// function signature
memoizeOne(fn /* function to memoize */, isEqual /*equality check for params*/)

// What we use
memoizeOne(fn)
```